### ISSUE TITLE
atom/button: disabled button of type link still works

### ISSUE THREAD

**Issue by midudev** [SUI-Components/sui-components#387]
When a button is disabled, we expect to avoid following the link if the button is of link type. Instead of this, even when the layout is disabled, it still is a functional link.

**Comment by JoanClaret** [comment]
@turolopezsanabria : disabled link buttons still redirecting, so we can't close this PR yet.

**Comment by oriolpuig** [comment]
I've tried to solve that issue and I think we can manage it adding a condition at href link attribute. I don't know if It won't be fixed at 100% but it is working at Chrome, Safari and Firefox browsers.

It would be something like code bellow:

```html
    <Link
      {...attrs}
      href={!disabled && href}
      target={target}
      rel={target === '_blank' ? 'noopener' : undefined}
    >
      {children}
    </Link>
```

What do you think about that?

**Comment by midudev** [comment]
Do a PR @oriolpuig!

**Comment by oriolpuig** [comment]
This issue has been fixed on PR #871
